pkg/loader/load.go

```diff
@@ -452,7 +452,7 @@ func (s *loaderImpl) processMysqlDDL(ddl *DDL) error {
 		return nil
 	})
 
-	if err != nil && isSetTiFlashReplica(ddl.SQL) {
+	if err != nil && isSetTiFlashReplicaOrTiFlashMode(ddl.SQL) {
```

Comment:
Maybe we should ignore all errors related with TiFlash? Since TiFlash statements are not available in other dbs.